Sources/Logging/Logging.swift

```diff
@@ -825,7 +825,7 @@ internal struct StdioOutputStream: TextOutputStream {
                 funlockfile(self.file)
                 #endif
             }
-            _ = fputs(ptr, self.file)
+            _ = fwrite(ptr, 1, strlen(ptr), self.file)
```

Comment:
That's not quite enough because this will also terminate on `0` bytes. Try for example (that's actually worth a unit test) logging this string `"hello\u{0} world"`. It'll only print the `hello` not the `world` I think.

The right fix is to change how we obtain the pointer from `withCString` to

```
var contiguousString = string
#if compiler(>=5.1)
contiguousString.makeContiguousUTF8()
#else
contiguousString = string + "" // this is the Swift 5.0 hack to make the string contiguous
#endif

contiguousString.utf8.withContiguousStorageIfAvailable { utf8Bytes /* this has a pointer and length */ in
   // exising code
}! /* the bang is safe because it's guaranteed contiguous */
```